Automattic/wp-calypso
# Domain Management i1: Add/domains table owners column

This PR adds owners column to the domains table and hide the column when domains fetched do not have owners associated with them.

Closes https://github.com/Automattic/dotcom-forge/issues/3561
![image](https://github.com/Automattic/wp-calypso/assets/47489215/018d02de-6eac-447f-b30e-6555bb006f7f)


## Proposed Changes

* Add owners column 
* Add util function `shouldHideOwnerColumn` that determine if column should be hidden or not

## Testing Instructions



* go to `/domains/manage?flags=domains/management`
* Verify owners column is shown
* Verify if the list of domains doesnt have an owner then column is not shown (can modify the mock-domains in BE to return undefine owners column, fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Sqbznvaf%2Sraqcbvagf%2Szbpxf%2Szbpx%2Qqbznvaf.cuc%3Se%3Qs0ro3o23%23125-og)

## Pre-merge Checklist
